libs/pages/login/src/lib/feature/page-logout-feature/page-logout-feature.spec.tsx

```diff
@@ -1,9 +1,23 @@
-import PageLogoutFeature from './page-logout-feature'
 import { render } from '__tests__/utils/setup-jest'
+import posthog from 'posthog-js'
+import React from 'react'
+import PageLogoutFeature from './page-logout-feature'
+
+const PostHogWrapper = ({ children }: { children: React.ReactElement }) => {
```

Comment:
I understand better for Posthog ! 
Did you apply it everywhere you use it?

Reply:
yes